### ISSUE TITLE
KopiaUI change date format to ISO standard/detect system locale

### ISSUE THREAD

**Issue by MildlyInterested** [kopia/kopia#2798]
1. Open KopiaUI.
2. Go to Snapshots and click on a path.
3. See list of snapshots and their start time.
4. Be confused/mildly annoyed because you're not american and do not use the mm/dd/yyyy format.

Solution:
- Detect system locale
- Have additional setting in the ui-preferences.json to set it there
- Failing above defaulting to ISO 8601 because that is globally understood and has no ambiguity

**Comment by andriej** [comment]
Windows 10:
![image](https://user-images.githubusercontent.com/6303489/224291483-116e80ab-eaa0-44dc-87d0-4d77622c0b79.png)

I do not see the issue on my client

**Comment by iav** [comment]
Duplicate of #1572

**Comment by tio-trom** [comment]
I agree. It is way easier to read the day/month/year when you see the backups in line as such:

![2023-11-08_00-54](https://github.com/kopia/kopia/assets/38043315/ed131afd-8ea0-4d0f-a525-f30c420d4f6d)

In my case it is confusing since the snapshot time always starts with the month instead of with the day.

**Comment by darrencoen** [comment]
This was closed as "not planned" in #1572? Is that still the status of this open ticket?
